release-notes/opensearch-release-notes-1.3.20.md

```diff
@@ -0,0 +1,117 @@
+# OpenSearch and OpenSearch Dashboards 1.3.20 Release Notes
+
+## Release Details
+
+[OpenSearch and OpenSearch Dashboards 1.3.20](https://opensearch.org/versions/opensearch-1-3-20.html) includes the following bug fixes, infrastructure and maintenance updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-1.3.20.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-1.3.20.md).
```

Comment:
Can we confirm these are available already?

Reply:
Yes. I can confirm these release notes are already merged.